filebeat/module/logstash/_meta/kibana/7/dashboard/Filebeat-logstash-slowlog.json

```diff
@@ -479,14 +479,14 @@
                     {
                         "col": 1, 
                         "columns": [
-                            "logstash.slowlog.level", 
+                            "log.level", 
                             "logstash.slowlog.plugin_type", 
                             "logstash.slowlog.plugin_name", 
                             "logstash.slowlog.message", 
```

Comment:
@ycombinator Just saw this field here. I assume we did not map this to ECS on purpose?